pyoverkiz/const.py

```diff
@@ -16,86 +17,92 @@
 SOMFY_CLIENT_ID = "0d8e920c-1478-11e7-a377-02dd59bd3041_1ewvaqmclfogo4kcsoo0c8k4kso884owg08sg8c40sk4go4ksg"
 SOMFY_CLIENT_SECRET = "12k73w1n540g8o4cokg0cw84cog840k84cwggscwg884004kgk"
 
+SERVERS_THAT_SUPPORT_LOCAL_API = [
```

Comment:
What about?

```suggestion
SERVERS_W_LOCAL_API = [
```

Reply:
I am not a huge fan of abbreviations like `W`, what about `SERVERS_WITH_LOCAL_API`? Or `DEV_MODE_SERVERS`?